website/docs/r/encryption_at_rest.html.markdown

```diff
@@ -59,13 +59,56 @@ resource "mongodbatlas_encryption_at_rest" "test" {
 
 **NOTE**  if using the two resources path for cloud provider access, `cloud_provider_access_setup` and `cloud_provider_access_authorization`, you may need to define a `depends_on` statement for these two resources, because terraform is not able to infer the dependency.
 
-```hcl
+```terraform
 resource "mongodbatlas_encryption_at_rest" "default" {
   (...)
   depends_on = [mongodbatlas_cloud_provider_access_setup.<resource_name>, mongodbatlas_cloud_provider_access_authorization.<resource_name>]
 }
 ```
 
+## Example of encryption at rest with azure and a cluster using its encryption at rest
+
+**NOTE** Since the cluster wants to use the encryption at rest it should wait for `mongodbatlas_encryption_at_rest` to finish first by using implicit dependency of `project_id`(see the example below).
```

Comment:
Slight edit, I'd remove Note as this is is really general instructions for the example:
The configuration of encryption at rest with customer key management, `mongodbatlas_encryption_at_rest`,  needs to be completed before a cluster is created in the project.   Force this wait by using an implicit dependency via `project_id` as shown in the example below.